packages/newspringchurchapp/android/app/build.gradle

```
@@ -146,7 +146,7 @@ android {
         applicationId "cc.newspring.newspringapp"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 25
+        versionCode 23
```

@redreceipt There are a few places (like this one) where the version number went down. They should all go up, right? I'm getting a downgrade error on Android for sure even though iOS works. I think we should bump them up. 
```suggestion
        versionCode 26
```